Ticket-booking-spring-app/src/main/java/com/example/demo/service/room/RoomService.java

```diff
@@ -0,0 +1,9 @@
+package com.example.demo.service.room;
+
+import com.example.demo.entity.Room;
+
+import java.util.Optional;
+
+public interface RoomService {
```

Comment:
Ten interfejs (i inne) są potrzebne? :)

Reply:
Z tymi interfejsami wzorowałam się na kursie Springa, z którego się uczyłam. Dzięki temu dokładniej widać jakie metody udostępnia serwis nie wczytując się w implementację. Ale być może nie jest to wystarczający powód żeby wprowadzać tyle dodatkowych interfejsów ;)